src/Ledger/Gov.lagda

```diff
@@ -9,9 +9,9 @@ open import Data.List.Relation.Unary.Any
   renaming (Any to Anyˡ; any? to decAny)
 open import Function.Related using (fromRelated)
 open import Function.Related.Propositional using (⤖⇒)
+open import Relation.Binary using (Rel)
 open import Relation.Nullary.Decidable renaming (map to mapᵈ)
-
-open import Ledger.Prelude renaming (yes to yesᵈ; no to noᵈ)
+open import Ledger.Prelude renaming (yes to yesᵈ; no to noᵈ) hiding (Rel)
```

Comment:
Please restore this to the original. `Rel` is unused in this file, so there's no reason to change up the imports.

Reply:
I'm sure it was used at some point, otherwise I certainly wouldn't have changed the imports.  It must have gone away recently... anyway, I'll gladly change the imports back. 👍🏼